.gitignore

```diff
@@ -2,3 +2,113 @@
 *.pyc
 local_settings.py
 *.db
+
```

Comment:
**praise**: Thanks for including this. As this was a project that I used to be the only developer, this didn't bothered me much, but when we have new contributors, it is very helpful to make git easier to use.